packages/renderer/src/lib/image/ImageDetailsFiles.svelte

```diff
@@ -27,9 +28,15 @@ onMount(async () => {
   filesProvidersUnsubscribe = imageFilesProviders.subscribe(providers => {
     if (providers.length === 1 && imageInfo) {
       filesProvider = providers[0];
-      window.imageGetFilesystemLayers(filesProvider.id, imageInfo).then(layers => {
-        imageLayers = layers;
-      });
+      loading = true;
+      window
+        .imageGetFilesystemLayers(filesProvider.id, imageInfo)
+        .then(layers => {
```

Comment:
should we handle potential errors ?

Reply:
Yes, this is also my wondering, I'll create a specific issue for this